fiftyone/operators/executor.py

```diff
@@ -648,6 +651,11 @@ def ops(self):
         """
         return self._ops
 
+    @property
+    def user_id(self) -> Optional[str]:
+        """The ID of the user executing the operation."""
```

Comment:
Might recommend adding `, if known` to the docstring to reinforce that this is not always available.